core/src/main/java/org/elasticsearch/tribe/TribeService.java

```diff
@@ -154,6 +156,12 @@ public static Settings processSettings(Settings settings) {
     public static final Set<String> TRIBE_SETTING_KEYS = Sets.newHashSet(TRIBE_NAME_SETTING.getKey(), ON_CONFLICT_SETTING.getKey(),
             BLOCKS_METADATA_INDICES_SETTING.getKey(), BLOCKS_METADATA_SETTING.getKey(), BLOCKS_READ_INDICES_SETTING.getKey(), BLOCKS_WRITE_INDICES_SETTING.getKey(), BLOCKS_WRITE_SETTING.getKey());
 
+    // these settings should be passed through to each tribe client, if they are not set explicitly
+    private static final List<String> PASS_THROUGH_SETTINGS = Arrays.asList(
+        NetworkService.GLOBAL_NETWORK_HOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_BINDHOST_SETTING.getKey(),
+        NetworkService.GLOBAL_NETWORK_PUBLISHHOST_SETTING.getKey()
```

Comment:
I will add the transport setting equivalents to the whitelist. As @clintongormley explained, we would like to stick with a whitelist as it allows a completely controlled environment and no surprises. There are a limited set of these settings that make sense, and we will add something in the future to allow plugins to opt in for passing through.